snappy_wrappers/wrappers/gcnv/contig_ploidy/wrapper.py

```diff
@@ -43,14 +43,24 @@
 export OMP_NUM_THREADS={snakemake.threads}
 export THEANO_FLAGS="base_compiledir=$TMPDIR/theano_compile_dir"
 
+# Get contig name style
+grep "SN:chr" {snakemake.input.interval_list} > /dev/null && true
+exit_status=$?
+if [[ $exit_status == 0 ]]; then
+    STYLE="chr"
+else
+    STYLE=""
+fi
+
+
```

Comment:
I don't know what the input interval_list looks like; I assume/hope it does not mix "chr" and "" prefixes ;)
Personally, I would try to make sure that the "SN:chr" is a bit more specific (i.e. start of line, followed by a number or X/Y and/or has trailing newline/tab/space/whatever).

Reply:
I've specified the regex, and I think if the file would have mixed prefixes a lot of other things would also break, so I'm not too concerned about check for that here.